Author: Sven Joachim <svenjoac@gmx.de>
Description: Debian changes to rxvt* terminfo entries
 Change backspace key description according to the Debian keyboard policy.
 .
 Provide rxvt-m as an alias to rxvt-basic, because the Debian rxvt
 package used to set $TERM to rxvt-m on monochrome displays.
Bug-Debian: https://bugs.debian.org/1037071
Forwarded: not-needed
Last-Update: 2023-07-05

---
 misc/terminfo.src |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/misc/terminfo.src
+++ b/misc/terminfo.src
@@ -6721,7 +6721,7 @@ mlterm-256color|mlterm 3.0 with xterm 25
 #
 # removed dch/dch1 because they are inconsistent with bce/ech -TD
 # remove km as per tack test -TD
-rxvt-basic|rxvt terminal base (X Window System),
+rxvt-basic|rxvt-m|rxvt terminal base (X Window System),
 	OTbs, am, bce, eo, mir, msgr, xenl, xon, XT,
 	cols#80, it#8, lines#24,
 	acsc=``aaffggjjkkllmmnnooppqqrrssttuuvvwwxxyyzz{{||}}~~,
@@ -6733,7 +6733,7 @@ rxvt-basic|rxvt terminal base (X Window
 	enacs=\E(B\E)0, flash=\E[?5h$<100/>\E[?5l, home=\E[H,
 	ht=^I, hts=\EH, ich=\E[%p1%d@, il=\E[%p1%dL, il1=\E[L,
 	ind=\n, is1=\E[?47l\E=\E[?1l,
-	is2=\E[r\E[m\E[2J\E[H\E[?7h\E[?1;3;4;6l\E[4l, kbs=^H,
+	is2=\E[r\E[m\E[2J\E[H\E[?7h\E[?1;3;4;6l\E[4l, kbs=^?,
 	kcbt=\E[Z, kmous=\E[M, rc=\E8, rev=\E[7m, ri=\EM, rmacs=^O,
 	rmir=\E[4l, rmkx=\E>, rmso=\E[27m, rmul=\E[24m,
 	rs1=\E>\E[1;3;4;5;6l\E[?7h\E[m\E[r\E[2J\E[H,
